packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/DisclosureComponent.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import Button from 'terra-button';
+import ContentContainer from 'terra-content-container';
+import TextField from 'terra-form/lib/TextField';
```

Comment:
This can be replaced with terra-form-input's InputField component.

Reply:
I told Derek I would update this and promptly forgot. I'll get these switched over.